vello_encoding/src/lib.rs

```diff
@@ -32,6 +32,8 @@
 #![allow(
     unnameable_types,
     clippy::shadow_unrelated,
+    // Only applies on WASM?
+    variant_size_differences,
```

Comment:
It is more likely that it is 32 bit targets, not just wasm, like I ran into in Peniko. But some 32 bit targets can't do AtomicU64, and so we don't support building on them all.